readme.md

```diff
@@ -73,7 +73,7 @@ Class with a `enqueue` and `dequeue` method, and a `size` getter. See the [Custo
 
 #### .add(fn, [options])
 
-Returns the promise returned by calling `fn`.
+Returns the promise or value returned by calling `fn`. This allows adding both sync and async tasks to the queue.
```

Comment:
@onury I don't think you should have to be aware what `fn` returns when doing `add`. What do you think about `add()` always returning a promise, for consistency?

Reply:
How about;
> _Allows adding a sync or async task to the queue. Always returns a promise._